README.md

```diff
@@ -1,30 +1,48 @@
 # Kyma Community
 
-Welcome to the Kyma community.
-
 ## Overview
 
-This is an open repository for all Kyma contributors.
+Welcome to the Kyma community. This repository is your starting point for contributing code and content to Kyma.
+
+Read the document to learn:
+- which rules govern the Kyma community
+- how to submit contributions that follow the recommended flow, approved templates, and standards
+- who to contact for questions or suggestions
+
+## Prerequisites
+
+Before you learn how to contribute, read the [Code of Conduct](CODE_OF_CONDUCT.md) for rules of behavior for all contributors and maintainers in the community. Also see the Kyma [governance](governance.md) model to better understand the role of a maintainer and the process of becoming one.
```

Comment:
`read the [Code of Conduct](CODE_OF_CONDUCT.md) document to learn the rules of behavior`?

- to avoid double "for"